scripts/ci.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -xe
+
+if [ ! -z "$1" ]
+then
+    report_location=$1
+else
+    report_location=`pwd`
+fi
+
+./gradlew --no-daemon --info clean
+./gradlew --no-daemon --info build countDebugDexMethods -PdisablePreDex
+if [ -n "${GCLOUD_SERVICE_KEY+1}" ]; then
```

Comment:
Ah, good call - @SachseInTheHub did a nice job of this in another project - perhaps you can share?